db-engine/src/main/java/com/splicemachine/db/impl/sql/misc/sqlcommentstrip.jj

```diff
@@ -0,0 +1,89 @@
+options
+{
+	STATIC = false;
+	LOOKAHEAD = 1;
+	DEBUG_PARSER = false;
+	DEBUG_LOOKAHEAD = false;
+	DEBUG_TOKEN_MANAGER = false;
+	ERROR_REPORTING = true;
+	USER_TOKEN_MANAGER = false;
+	USER_CHAR_STREAM = true;
+	CACHE_TOKENS = true;
+	UNICODE_INPUT = true;
+}
+
+PARSER_BEGIN(SQLCommentStripper)
+
+package com.splicemachine.db.impl.sql.misc;
+
+import com.splicemachine.db.impl.sql.misc.ParseException;
+import com.splicemachine.db.impl.sql.misc.Token;
+import com.splicemachine.db.impl.sql.misc.TokenMgrError;
+import com.splicemachine.db.impl.sql.misc.SQLCommentStripperConstants;
+import com.splicemachine.db.impl.sql.misc.CharStream;
+
+import com.splicemachine.db.iapi.error.StandardException;
+
+public class SQLCommentStripper {
+
+}
+
+PARSER_END(SQLCommentStripper)
+
+TOKEN_MGR_DECLS :
+{
+	int commentNestingDepth = 0;
+}
+
+MORE : {<"/*"> {commentNestingDepth = 1;} : IN_COMMENT}
+
+<IN_COMMENT> MORE :
+{
+    <"/*"> {commentNestingDepth +=1;}
+}
+
+<IN_COMMENT> SKIP :
+{
+    <"*/">
+    {
+        commentNestingDepth -= 1;
+        SwitchTo(commentNestingDepth==0 ? DEFAULT : IN_COMMENT);
+    }
+}
+
+<IN_COMMENT> MORE :
+{
+    < ~[] >
+}
+
+TOKEN :
+{
+    <OPEN_QUOTE: "'"> : IN_QUOTATION
+}
```

Comment:
As strange as it may seem, we could have a table name or column name of "/*", so we should probably parse out double quotes as well so such weird names aren't stripped out or cause the javaCC parser to choke.

Reply:
Thanks @msirek for catching this. I think I missed that scenario, I'll work on a fix for it.

Reply:
Hi @msirek I've pushed a commit to take into consideration a statement like "table_name/*". Test is updated too. Thanks.

Reply:
Thanks @yxia92, looks good.